vmm/src/memory_manager.rs

```diff
@@ -1478,9 +1477,29 @@ impl MemoryManager {
         let page_size = 4096; // TODO: Does this need to vary?
         let mut table = MemoryRangeTable::default();
         for r in &self.guest_ram_mappings {
-            let dirty_bitmap = self.vm.get_dirty_log(r.slot, r.size).map_err(|e| {
+            let kvm_dirty_bitmap = self.vm.get_dirty_log(r.slot, r.size).map_err(|e| {
```

Comment:
Can you please retain the original name `dirty_bitmap`? If you want to avoid rebinding `dirty_bitmap` later, you can also name this `dirty_log` to match `get_dirty_log`. Another option is to name it `vm_dirty_bitmap`.

Reply:
Sure. Renamed to `vm_dirty_bitmap` (from `kvm_dirty_bitmap`) and `vmm_dirty_bitmap` (from `vm_memory_dirty_bitmap`).